defaults/main.yml

```diff
@@ -395,4 +372,46 @@ checkmk_agent__git_dest: '{{ "/usr/local/src/check-mk/" + checkmk_agent__git_rep
 checkmk_agent__git_version: 'auto'
                                                                    # ]]]
                                                                    # ]]]
+# Configuration for other Ansible roles [[[
+# -----------------------------------------
+
+# .. envvar:: checkmk_agent__apt_preferences__dependent_list [[[
+#
+# Configuration for the debops.apt_preferences_ role.
+checkmk_agent__apt_preferences__dependent_list:
+
+  - package: 'check-mk-agent'
+    backports: [ 'jessie' ]
+    reason: 'Package not available in stable Debian Jessie'
+    by_role: 'debops-contrib.checkmk_agent'
+
+                                                                   # ]]]
+# .. envvar:: checkmk_agent__ferm__dependent_rules [[[
+#
+# Configuration for the debops.ferm_ role.
+checkmk_agent__ferm__dependent_rules:
+
+  - type: 'accept'
+    dport: [ 'check-mk-agent' ]
+    saddr: '{{ checkmk_agent__allow }}'
+    accept_any: True
+    weight: '20'
+    by_role: 'debops-contrib.checkmk_agent'
+    rule_state: '{{ "xinetd" in checkmk_agent__type | ternary("present", "absent") }}'
+
+                                                                   # ]]]
+# .. envvar:: checkmk_agent__tcpwrappers__dependent_allow [[[
+#
+# Configuration for the debops.tcpwrappers_ Ansible role.
+checkmk_agent__tcpwrappers__dependent_allow:
+
+  - daemon: 'inetd'
```

Comment:
@drybjed Thanks but I tried all that. As I said above, it seems that xinetd itself checks the particular service again. So two entries are needed.